recipes/torchapp/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "torchapp" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchapp-{{ version }}.tar.gz
+  sha256: 8084f005dbd3548aa1ffb341b6539325adc475b5d9cfcb5967d4af45ab2e9ff2
+
+build:
+  entry_points:
+    - torchapp = torchapp.main:app
+    - torchapp-imageclassifier = torchapp.examples.image_classifier:ImageClassifier.main
+  skip: true  # [win]
```

Comment:
If the reason for skipping Windows is a missing dependency, we still prefer to make this noarch (if possible) and let Windows fail. That means this package would be ready when the missing dependency is available without the need for any changes here.

Reply:
@ocefpaf thank you for the review. This is the error I get with Windows:

```
conda_libmamba_solver.exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("pytorch[version='>=1.12.1']"), MatchSpec("torchvision[version='>=0.13.0']"), MatchSpec("fastai==2.7.10=pyhd8ed1ab_0"), MatchSpec(">=1.7,<1.14")}
Encountered problems while solving:
  - nothing provides requested pytorch >=1.12.1
  - nothing provides requested torchvision >=0.13.0
  - nothing provides pytorch >=1.7,<1.14 needed by fastai-2.7.10-pyhd8ed1ab_0
```

Could I revert the PR back to [`7dad632`](https://github.com/conda-forge/staged-recipes/pull/24165/commits/7dad632d28f0c05b2417d2a5fb3d3a15ba0f4ae2) then?

Reply:
👍

Reply:
@ocefpaf done. Please merge when ready. Thanks!